internal/storages/s3/session.go

```diff
@@ -96,3 +98,10 @@ func createSession(bucket string, settings map[string]string) (*session.Session,
 
 	return session.NewSession(config)
 }
+
+func loggedClose(c io.Closer) {
```

Comment:
If you want to pass a message to LoggedCLose then change the signature of the function, don't copy it. https://en.wikipedia.org/wiki/Don%27t_repeat_yourself